src/umessage/umessagebuilder.rs

```diff
@@ -231,21 +225,21 @@ impl UMessageBuilder {
     /// # Examples
     ///
     /// ```rust
-    /// use up_rust::{UMessageBuilder, UMessageType, UPayloadFormat, UPriority, UUIDBuilder, UUri};
-
+    /// use up_rust::{UMessageBuilder, UMessageType, UPayloadFormat, UPriority, UUri};
     ///
     /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
     /// let invoked_method = UUri::try_from("my-vehicle/cabin/1/rpc.doors")?;
     /// let reply_to_address = UUri::try_from("my-cloud/dashboard/1/rpc.response")?;
-    /// let request_id = UUIDBuilder::build();
-    /// let message = UMessageBuilder::response(reply_to_address.clone(), request_id.clone(), invoked_method.clone())
-    ///                    .with_message_id(UUIDBuilder::build())
+    /// # let request_msg = UMessageBuilder::request(invoked_method.clone(), reply_to_address.clone(), 1000).build();
+    /// // extracting reqid from received Request message for use in building Response message
+    /// let reqid = request_msg.as_ref().unwrap().clone().attributes.as_ref().unwrap().id.as_ref().unwrap().clone();
```

Comment:
I'm sorry, but FMPOV this only confuses the reader because
* the code won't compile if copied from docs to local workspace (no request_msg) and
* the code for extracting the reqid looks terrible ;-)

Can't we simply create a reqid using the UUIDBuilder and then add some wording indicating that a service implementation would usually use `response_for_request` instead? FMPOV this would make things much clearer and more explicit ...

Reply:
I uhhh, will grant you point 2. 😅

Okay, I think you're right, having a copy-pasteable example is good and a comment can explain what we would normally do.